public/app/sass/_component.appspage.scss

```diff
@@ -34,8 +35,18 @@ p.apps-text > strong {
 a.plan-route-button
 {
   display:inline-block;
-  width:300px;
+  width:350px;
   height:100px;
+  background: rgb(191,41,41) url(../../images/logo.svg) left no-repeat;
+  background-size: 80px;
+  background-position:10px 20px;
+  font-family:"PT Sans";
+  color:#FFFFFF;
```

Comment:
@pietercolpaert  I think it's a good practice that the color variable is lowercase'ed and in a variable.